netlify.toml

```diff
@@ -0,0 +1,12 @@
+[build]
+  publish = "dist"
```

Comment:
I think standard TOML indentation is to keep the key/value lines flush with the header (even when a whole section block is indented, as you have below).

Reply:
da6f456